packages/documentation/src/stories/components/segmented-button/segmented-button.stories.ts

```diff
@@ -79,3 +79,28 @@ export const IconExample: Story = {
     `;
   },
 };
+
+export const TextAndIconExample: Story = {
+  render: (args: Args) => {
+    const labelCount = Math.min(args.labelCount || 0, MAX_LABELS);
+    const labelsArray = Array.from({ length: labelCount }, (_, i) => `Label ${i + 1}`);
+    const name = `segmented-button-${Math.random().toString(36).slice(-6)}`;
+
+    return html`
+      <div class="segmented-button-container">
+        <fieldset class="segmented-button">
+          <legend>Choose one of the options</legend>
+          ${labelsArray.map(
+            (label, index) => html`
+              <label class="segmented-button-label">
+                <input type="radio" name="${name}" checked="${index === 0 ? '' : nothing}" />
+                <post-icon name="${1000 + index}"></post-icon>
+                ${label}
+              </label>
+            `,
+          )}
+        </fieldset>
+      </div>
+    `;
+  },
```

Comment:
There renderer is duplicated for the `TextExample`, the `IconExample`, and the `TextAndIconExample` stories. Can you refactor to have a generic renderer and ad it to the [meta object](https://github.com/swisspost/design-system/blob/dafe1feecd8108d2e20ce24606dbbe961bee2d0e/packages/documentation/src/stories/components/segmented-button/segmented-button.stories.ts#L7) instead ?